functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/newtests/HistoricalChangesNewTestsSelectionExecutionFunctionalTest.java

```diff
@@ -47,7 +51,9 @@ public void should_only_execute_new_test_when_multiple_commits_with_changed_test
 
         // when
         final List<TestResult> actualTestResults = project
-            .withEnvVariables("git.last.commits", "3")
+            .buildOptions()
+                .withEnvVariables("git.last.commits", "3")
```

Comment:
I know the convention, but if we are using the fully qualified form in all methods, why not rename this to `withEnvironmentVariables`. I think that in case of system properties we would call it `withSystemProperties` and not `withSysProperties`